packages/lesswrong/lib/collections/tags/schema.ts

```diff
@@ -188,6 +189,44 @@ export const schema: SchemaType<DbTag> = {
     canCreate: ['admins', 'sunshineRegiment'],
     ...schemaDefaultValue(false),
     group: formGroups.advancedOptions
+  },
+
+  tagFlagsIds: {
+    ...arrayOfForeignKeysField({
+      idFieldName: "tagFlagsIds",
+      resolverName: "tagFlags",
+      collectionName: "TagFlags",
+      type: "TagFlag",
+    }),
+    control: 'TagFlagToggleList',
+    label: "Flags: ",
+    optional: true,
+    order: 30,
+    viewableBy: ['guests'],
+    editableBy: ['members', 'sunshineRegiment', 'admins'],
+    insertableBy: ['sunshineRegiment', 'admins']
```

Comment:
What does it mean for this to be editable by members, but insertable only by sunshine/admins? Only sunshine/admins can add to the list, but anyone can remove from the list?

Reply:
Ah, no. What this does is that you cannot create an element with flags, but you can edit any existing tag to have flags. 

The idea was that I didn't want the flag component to clog up the form for creating a new tag, but I did want to make it available to everyone to add flags to a tag that already exists. We could also solve this at the fragment level, but doing it at the permission level was a bit easier.